okio-behavior/src/main/java/okio/test/ByteRate.java

```diff
@@ -0,0 +1,59 @@
+package okio.test;
+
+import java.util.concurrent.TimeUnit;
+
+/** A byte count expressed over a period of time for determining transfer rates. */
+public final class ByteRate {
```

Comment:
We could add a dependency to [ByteUnits][1] :grinning: 
```java
rate.count(28, KIBIBYTES).per(5, SECONDS);
rate.count(28, MEGABYTES).per(5, SECONDS);
```
[1]: https://github.com/JakeWharton/byteunits